pom.xml

```diff
@@ -98,7 +99,7 @@
         <jsr305.version>3.0.2</jsr305.version>
         <really.executable.jar.version>1.5.0</really.executable.jar.version>
         <generext.version>1.0.2</generext.version>
-        <avro.random.generator.version>0.2.1</avro.random.generator.version>
+        <avro.random.generator.version>0.2.2</avro.random.generator.version>
```

Comment:
This version bump is needed in order to pick up [this change](https://github.com/confluentinc/avro-random-generator/pull/8), or else sometimes a NullPointerException is encountered during data generation, when running benchmarks with multiple threads.